File: platforms/swift/examples/core/texture_mip_chain/levels.swift
import FragmentColor

let pixels = Array(repeating: 0, count: 8 * 8 * 4)
let chain = try TextureMipChain.prepare((
    pixels,
    TextureFormat.rgba8UnormSrgb,
    [8, 8],
))
let level_zero_bytes = chain.levels()[0]
let _ = level_zero_bytes

File: platforms/python/examples/core/texture_mip_chain/levels.py
from fragmentcolor import TextureFormat, TextureMipChain

pixels = [0] * (8 * 8 * 4)
chain = TextureMipChain.prepare(pixels, TextureFormat.Rgba8UnormSrgb, [8, 8])
level_zero_bytes = chain.level(0)
_ = level_zero_bytes
